okapi-core/src/main/java/org/folio/okapi/util/ProxyContext.java

```diff
@@ -203,20 +204,20 @@ public MultiMap getHandlerHeaders() {
   /**
    * Log that HTTP request has been received.
    * @param ctx routing context
-   * @param tenant tenant
    */
-  public final void logRequest(RoutingContext ctx, String tenant) {
+  public final void logRequest(RoutingContext ctx) {
     Timer.Sample sample = MetricsHelper.getTimerSample();
-    StringBuilder mods = new StringBuilder();
-    if (modList != null && !modList.isEmpty()) {
-      for (ModuleInstance mi : modList) {
-        mods.append(" ").append(mi.getModuleDescriptor().getId());
-      }
+    String mods = "";
+    if (modList != null) {
+      mods = modList.stream().map(x -> x.getModuleDescriptor().getId())
+          .collect(Collectors.joining(" "));
     }
```

Comment:
The mods calculation should be moved into the isInfoEnabled() if clause.

Reply:
Right. The modules are separated by space. Would it be better to use , ?